docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Manging the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Install package controller after installation
+
+{{% alert title="Important" color="warning" %}}
+**Recommendation**: Ideally, you should never delete package controller after installation. When the package controller is re-installed, it will try to pull and activate the latest bundle. So, there is a risk that your packages will be upgraded to a different version. See more information about package bundles [here]({{< relref "./packagebundles" >}})
+{{% /alert %}}
```

Comment:
Installed packages won't be automatically upgraded even if a different bundle gets activated in the pbc resource. Actually, the risk is if the pbc resource is deleted and then gets recreated, the active bundle recorded in the pbc resource will point to the latest bundle.

Reply:
but you can always get back to the previous bundle version by running `eksctl anywhere upgrade packages --bundle-version xxx --cluster xxx`.